packages/headless/src/extensions/slash-command.tsx

```diff
@@ -28,7 +29,7 @@ const Command = Extension.create({
   },
 });
 
-const renderItems = () => {
+const renderItems = (elementRef: RefObject<Element> | null = null) => {
```

Comment:
I think you need to make the elementRef optional
`elementRef?: RefObject<Element> | null = null`
I just want to be sure that TS won't complain about missing argument when calling renderItems()

Reply:
@andrewdoro I've made the requested changes in the same branch. Do I need to create another PR?